payu/models/cice.py

```diff
@@ -221,22 +256,17 @@ def setup(self):
             prior_setup_nml = f90nml.read(prior_nml_path)['setup_nml']
 
             # The total time in seconds since the beginning of the experiment
-            total_runtime = prior_setup_nml['istep0'] + prior_setup_nml['npt']
-            total_runtime = total_runtime * prior_setup_nml['dt']
-        else:
-            # Locate and link any restart files (if required)
-            if not setup_nml['ice_ic'] in ('none', 'default'):
-                self.link_restart(setup_nml['ice_ic'])
-
-            if setup_nml['restart']:
-                self.link_restart(setup_nml['pointer_file'])
```

Comment:
Why are these removed ?

Reply:
Ah yep, it's not very clear in the diff but these have just been moved up to L198-204 to make them run outside of the new `_calc_runtime()` method.

Reply:
Oh great :)